CONTRIBUTING.rst

```diff
@@ -53,16 +53,18 @@ alterations with ``Lint:`` and other non-PEP meta changes, such as updates to
 the Readme/Contributing Guide, issue/PR template, etc., with ``Meta:``.
 
 
-Sign the CLA
-------------
+Sign the Contributor License Agreement
+--------------------------------------
 
-Before you hit "Create pull request", please take a moment to ensure that this
-project can legally accept your contribution by verifying you have signed the
+All contributors need to sign the 
 `PSF Contributor Agreement <https://www.python.org/psf/contrib/contrib-form/>`_.
+to ensure we legally accept your work.
 
-If you haven't signed the CLA before, please follow the
-`steps outlined in the CPython devguide
-<https://devguide.python.org/pullrequest/#licensing>`_ to do so.
+If you haven't signed the CLA before,
+simply submit your Pull Request and the CLA bot will reply with instructions.
```

Comment:
```suggestion
submit your pull request and the CLA bot will reply with instructions.
```

Skip "simply", it's always simple until it isn't:

* https://www.knowledgeowl.com/blog/posts/dont-say-simply-jim-fisher/
* https://www.youtube.com/watch?v=gsT2BBWBVmM

I know this simply applies to opening a PR, but the CLA bot is meant to be simple too but we can ask Adam about that :)

And pull request is a common noun.

* https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests

Reply:
Yeah, I see the concern around "simply", though IMO its still worth making the point that the user should just submit their PR and the bot will do the rest. What about:

```suggestion
just submit your pull request and the CLA bot will reply with instructions.
```

Reply:
"Just" falls into the same category, I think we can safely leave it out and the meaning is still clear:

<img src=https://user-images.githubusercontent.com/1324225/203546609-93436d05-6c08-4c79-97e7-af7b232b2b85.png width=50%>

* https://www.knowledgeowl.com/blog/posts/dont-say-simply-jim-fisher/
* https://youtu.be/gsT2BBWBVmM?t=835

But let's not worry about it too much, I'll leave it up to you :)

Reply:
Yeah, I definitely understand the overall concern, but IMO context is important here. "Just" is problematic when it is being used, like "simply", to imply that the action involved should be easy for the user, which come across as condescending and presumptive, and makes the user feel stupid if it isn't—e.g. "Just write a m4 macro in configure to disable the feature." Here, "just" just serves to emphasize that the writer thinks the task should be easy for the reader, and it could be replaced with "simply", "trivially", "merely", etc. with no change in meaning.

In this case, like in the previous sentence, the intended meaning isn't to emphasize that the task is simple, but instead to make clear that the reader doesn't have to take some other action (sign the CLA beforehand), but rather "just" submit the PR. Syntactically, it could still be eliminated, and it is not essential semantically, but in this usage some meaning is lost, since removing it subtly shifts the emphasis from not needing to sign the CLA to an imperative command to submit a PR, which doesn't read quite as clearly or flow quite as well to me.

I played around with rewriting it to 

```rst
you don't need to do so beforehand;
the CLA bot will reply with instructions once you submit your first pull request.
```

but that didn't put nearly as strong an emphasis on this being the preferred course of action, as opposed to _just_ another option alongside an implied/assumed one of pre-signing the CLA.

Any ideas here? Or am I overthinking this (probably...)?

Reply:
Let's go for "just" :)

Reply:
I would say don't use "simply", "just". We're writing instructions in already lengthly doc. The less the better for the reader.

Reply:
"Go ahead and create the PR. Our bot will ping you about signing the CLA if needed.

Reply:
Okay, I'll replace both lines with [this](https://github.com/python/peps/pull/2900/files#r1031011700)